server/testdb/postgres.go

```diff
@@ -107,7 +107,15 @@ func (td *postgresDB) ServerID() (id string, isNew bool, err error) {
 }
 
 func (td *postgresDB) Drop() error {
-	return dropTables(td, "test_runs", "tests", "server", "schema_migrations")
+	return dropTables(
+		td,
+		"transaction_steps",
+		"transactions",
+		"test_runs",
+		"tests",
```

Comment:
I forgot to add the environments table here, is it necessary?

Reply:
yes, it is. I'll add it